### ISSUE TITLE
Broken Installer for fedora [./install_release.sh]

### ISSUE THREAD

**Issue by MysteriousImmature** [Genymobile/scrcpy#5470]
**meson** package must be installed to run ./install_release.sh.

Solution: Modify installer to install it to avoid installation failure.
![Screenshot from 2024-11-11 02-55-58](https://github.com/user-attachments/assets/df22cd44-b715-4039-9788-0f033b25d430)

**Comment by laama1** [comment]
Yes, except on fedora it's quite difficult to get the requested libavformat package.
libavformat-free the default, does not seem to work with this.
